docs/settings.rst

```diff
@@ -141,6 +141,25 @@ Language Detection
     datetime.datetime(2015, 12, 4, 0, 0)
 
 
+Default Languages
++++++++++++++++++
+
+``DEFAULT_LANGUAGES``: It is a ``list`` of language codes in ISO 639 which is used as default 
+languages and are to be parsed. eg. "en":
+
+    >>> from dateparser import parse
+    >>> parse(settings={'DEFAULT_LANGUAGES': ["en", "es"]})
+
+
+Optional language detect
+++++++++++++++++++++++++
+
+``LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD``: defaults to ``0.5``. It is a ``float`` of minimum required confidence for the custom language detection.:
+
+    >>> from dateparser import parse
+    >>> parse(settings={'LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD': 0.5})
```

Comment:
I think example is not complete:

- we are lacking the string to parse
- setting has no effect without language detection function, right?

Reply:
Yes, settings have no effect without a language detection function. Should I mention that in docs?

Reply:
I meant to update this example so that the setting makes a difference. I think we already mention that in the docs.

Reply:
This should be more like:
```suggestion
    >>> parse('3 de marzo de 2020', settings={'LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD': 0.5}, detect_languages_function=detect_languages)
```

Reply:
Thanks, I have made the changes 👍🏻